docker-compose.yml

```diff
@@ -170,8 +170,7 @@ services:
   mysql:
     image: mysql:5.7
     environment:
-      MYSQL_ROOT_PASSWORD:
-    # --sql-mode=NO_ENGINE_SUBSTITUTION fixes random SQL issues that showed up after upgrading MySQL to v5.7.
+      MYSQL_ROOT_PASSWORD: # --sql-mode=NO_ENGINE_SUBSTITUTION fixes random SQL issues that showed up after upgrading MySQL to v5.7.
```

Comment:
Did you intentionally remove these line breaks? Seems like they shouldn't have been made?